src/constants/types.tsx

```diff
@@ -1,3 +1,83 @@
+export enum ItemSeparatorType {
+  MoreMenuList,
+  VODCollapsibleList,
+  VODVideosList,
+}
+
+export interface VODCategoriesState {
+  categories: VODCategory[];
+  loading: boolean;
+  chosenVideo?: VODVideo;
+  chosenCategory?: { category: VODCategory; subCategory: VODSubCategory };
+}
+
+export interface VODRobots {
+  index: string;
+  follow: string;
+  maxSnippet: string;
+  maxImagePreview: string;
+  maxVideoPreview: string;
+}
+
+export interface VODOgImage {
+  width: number;
+  height: number;
+  url: string;
+  type: string;
+}
+
+export interface VODYoastHeadJson {
+  title: string;
+  robots: VODRobots;
+  canonical: string;
+  og_locale: string;
+  og_type: string;
+  og_title: string;
+  og_url: string;
+  og_siteName: string;
+  og_image: VODOgImage[];
+  article_modified_time: string;
+  twitter_card: string;
```

Comment:
In a JSON response, the variables are with an underscore.
In our code the variables are camelCase 
i.e twitter_card becomes twitterCard...